ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml

```diff
@@ -0,0 +1,158 @@
+---
+# Implement your Post Workload deployment tasks here
+
+# Leave these as the last tasks in the playbook
+
+- agnosticd_user_info:
+    data:
+      ocp4_guid: "{{ guid }}"
+      ocp4_domain: "{{ guid }}{{ subdomain_base_suffix }}"
+      ocp4_ssh_user: "{{ student_name }}"
+      ocp4_password: "{{ student_password }}"
+      ocp4_bastion: "{{ hostvars[groups.bastions.0].ansible_hostname }}.{{ guid }}{{ subdomain_base_suffix }}"
+  when:
+  - student_name is defined
+  - groups.bastions is defined
+- set_fact:
+    ansible_hostname: "{{ hostvars[groups.bastions.0].ansible_hostname }}"
+  when: groups.bastions is defined
+- set_fact:
+    ocp4_info:
+      ocp4_guid: "{{ guid | default('guid') }}"
+      ocp4_domain: "{{ subdomain_base_suffix | default('subdomain_base_suffix')}}"
+      ocp4_ssh_user: "{{ student_name | default('ec2-user') }}"
+      ocp4_password: "{{ student_password | default('undefined')}}"
+      ocp4_bastion:
+        "{{ ansible_hostname | default('bastion') }}.{{ guid | default('guid') }}
+        {{ subdomain_base_suffix | default('subdomain_base_suffix') }}"
+# - when: student_name is defined
+#   name: "Saving OCP4 cluster info for bookbag deployment"
+#   copy:
+#     owner: "{{ student_name }}"
+#     content: |
+#       [OCP4]
+#       guid={{ guid }}
+#       domain={{ subdomain_base_suffix }}
+#       student_name={{ student_name }}
+#       bastion={{ hostvars[groups.bastions.0].ansible_hostname }}.{{ guid }}{{ subdomain_base_suffix }}
+#     dest: "/home/{{ student_name }}/cluster.info"
+#   become: true
+
+- name: deploy oadp lab bookbag
+  when:
+  - student_name is defined
+  - cloud_credentials_data_string is defined
+  block:
+  - name: Get Konveyor lab git
+    git:
+      repo: "{{ bookbag_repo }}"
+      dest: "{{ bookbag_dir }}"
+      update: yes
+  - name: create .aws dir
+    file:
+      owner: "{{ ansible_user }}"
+      state: directory
+      path: "{{ bookbag_dir }}/{{ bookbag_build_dir }}/.aws"
+  - name: create .aws/credentials file to noobaa s3
+    copy:
+      owner: "{{ ansible_user }}"
+      content: "{{ cloud_credentials_data_string }}"
+      dest: "{{ bookbag_dir }}/{{ bookbag_build_dir }}/.aws/credentials"
```

Comment:
does it mean  the secret is exposed to the student, and this workload can only be executed in an AWS sandbox?

Reply:
Hi,  yes this will be exposed to the student because that API and storage will be used in the lab as it currently is.
And you are right, it will need to be limited to AWS sandbox. 
If possible we can maybe change that in the future (but that would be on a TODO at any rate now).

Reply:
Ok, I misunderstood the question at first. 

I removed the .aws dir and the credentials save. This will cause only a minor change in the lab docs. 
This change will not influence the operation of the lab and showcasing of capabilities as such. 

Thanks for bringing this to my attention!